mitodl/open-discussions
# Updated UI and views to use AuthenticatedSite (#444)

Part of #444 

#### What's this PR do?
Updates open-discussions to use new `AuthenticatedSite` records. Also removes and MicroMasters-specific code/text.

#### How should this be manually tested?
- Create a new `AuthenticatedSite` record for MicroMasters (per the README changes in this PR). Set `OPEN_DISCUSSIONS_DEFAULT_SITE_KEY` to that record's key
- Ensure you can still authenticate with OD and that the links back to MM function correctly.
- Create a second site with a different key and title and test [this PR](https://github.com/mitodl/micromasters/pull/3834) for it.
